packages/paste-website/src/pages/experiences/artificial-intelligence/index.mdx

```diff
@@ -0,0 +1,315 @@
+  const [isOpen, setIsOpen] = React.useState(true);
+  const sidePanelId = useUID();
+  return (
+    <SidePanelContainer id={sidePanelId} isOpen={isOpen} setIsOpen={setIsOpen}>
+      <SidePanelWithContent />
+      <SidePanelPushContentWrapper>
+        <Box paddingTop="space100" paddingLeft="space100">
+          <SidePanelBadgeButton variant="neutral">
+            <ArtificialIntelligenceIcon decorative />
+            Ask an assistant
+          </SidePanelBadgeButton>
+        </Box>
+      </SidePanelPushContentWrapper>
+    </SidePanelContainer>
+  )`}
+/>
+
+#### Chat empty state
+
+<LivePreview scope={{Box, Heading, ArtificialIntelligenceIcon, ChatComposerContainer, ChatComposer, ChatComposerActionGroup, Button, SendIcon, ProductCLIIcon, ProductMessagingIcon, ProductSendGridIcon}}>
+  {`<Box width="100%" display="flex" justifyContent="center" flexDirection="column">
+  <Box display="flex" flexDirection="column" alignItems="center" rowGap="space40" width="100%" padding="space130">
+  <ArtificialIntelligenceIcon decorative size="sizeIcon100"/>
+<Heading as="h6" variant="heading10">How can I help?</Heading>
+  <Box width="100%">
+    <ChatComposerContainer variant="contained">
+      <ChatComposer
+        ariaLabel="input for sending a message to ai"
+        placeholder="Chat text"
+        config={{
+          namespace: "customer-chat",
+          onError: (e) => {
+            throw e;
+          },
+        }}
+      />
+      <ChatComposerActionGroup>
+        <Button variant="primary_icon" size="reset">
+          <SendIcon decorative={false} title="send chat message to ai" />
+        </Button>
+      </ChatComposerActionGroup>
+    </ChatComposerContainer>
+    </Box>
+</Box>
+<Box display="flex" flexDirection="row" flexWrap="wrap" justifyContent="center" rowGap="space40" columnGap="space40">
+    <Button variant="secondary" size="rounded_small"><ProductMessagingIcon decorative />How do I set up SMS?</Button>
+    <Button variant="secondary" size="rounded_small"><ProductSendGridIcon decorative />I’m unable to login to SendGrid due to 2FA</Button>
+    <Button variant="secondary" size="rounded_small"><ProductSendGridIcon decorative />How can I get in touch with SendGrid support?</Button>
+    <Button variant="secondary" size="rounded_small"><ProductCLIIcon decorative />How to uninstall Twilio CLI?</Button>
+    <Button variant="secondary" size="rounded_small"><ProductSendGridIcon decorative />How do I report spam?</Button>
+</Box>
+</Box>
+`}
+</LivePreview>
+
+### Loading 
+
+<Box marginBottom="space70">
+  <Callout variant='neutral'>
+      <CalloutHeading as="h5">More details about this pattern are coming soon!</CalloutHeading>
+      <CalloutText>We haven’t yet patternized a general Loading pattern. If you need more details and standard examples for your feature, please consider partnering with us to [contribute](https://paste.twilio.design/introduction/contributing/patterns) it.</CalloutText>
```

Comment:
issue(blocking): same issue with links inside callout here

Reply:
good catches with the callout links!